tpm_futurepcr/util.py

```diff
@@ -119,3 +120,22 @@ def find_mountpoint_by_partuuid(partuuid):
                          stdout=subprocess.PIPE)
     res.check_returncode()
     return res.stdout.splitlines()[0].decode()
+
+class KeyValueAction(argparse.Action):
+    def __init__(self, option_strings, dest, **kwargs):
+        return argparse.Action.__init__(self,
+                option_strings=option_strings,
+                dest=dest,
+                **kwargs)
```

Comment:
There's probably no need to explicitly define `__init__` if it just calls the parent function without doing anything extra.

I see the argparse docs say "The easiest way to ensure these attributes are defined is to call `Action.__init__`", but that would still happen if the constructor was inherited from parent. (It feels like that advice is more directed towards completely custom classes than just subclasses of Action.)

Reply:
I pulled this boilerplate from an example, but thinking about the Python object model, you are certainly right.